vendor/stmicroelectronics/nucleo-wl55jc-profile-as923.yaml

```diff
@@ -0,0 +1,12 @@
+supportsClassB: true
+classBTimeout: 250
+pingSlotPeriod: 512
```

Comment:
Ping slot period is at most 128 seconds

Reply:
Thats true, the correct value defined in the LoRaWAN device is 16seconds.
Im aligned to the enum list included in schema.json. And this list defines all values as windows interval of 30ms.
https://github.com/TheThingsNetwork/lorawan-devices/blob/ba7ceb4fd89b385928e19bb9d676c98682a8b8f9/lib/draft/schema.json#L100-L103

But you're right, it's not very clear and it would be better to put in this enum the values from 2^0 to 2^7.

Reply:
You're right about this, we fixed this. Please rebase on `master`